src/main/java/frc/robot/commands/manipulator/SetManipulatorSpeed.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.commands.manipulator;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.manipulator.Manipulator;
+
+public class SetManipulatorSpeed extends CommandBase {
+    Manipulator manipulator;
+    double speed;
+    public SetManipulatorSpeed(Manipulator manipulator, double speed){
+        this.manipulator = manipulator;
+        this.speed = speed;
+    }
+
+    private void init() {
+        addRequirements(manipulator);
+    }
+
+    public void execute() {
+        manipulator.inputs.velocityRadPerSec = speed;
```

Comment:
this will not do anything. inputs shows you what the motor is doing, so velocityRadPerSec is the speed the motor is saying it is going. if you set it to something else, nothing will happen aside from the occasional bug.

```suggestion
        manipulator.setMotorPercentage(speed);
```